public/components/DataSourceAlertsCard/DataSourceAlertsCard.tsx

```diff
@@ -53,14 +54,22 @@ export const DataSourceAlertsCard: React.FC<DataSourceAlertsCardProps> =  ({ get
   const createAlertDetailsHeader = useCallback((alert) => {
     const severityColor = getSeverityColor(alert.severity);
     const triggerName = alert.trigger_name ?? DEFAULT_EMPTY_DATA;
+    const monitorUrl = `alerting#/monitors/${
```

Comment:
I think we should drop the `alerting` part since the app id varies with and without new home page settings

Reply:
Without the alerting part the URL is just appended to the all_overview page like this
`http://localhost:5601/w/HMDqJO/app/all_overview#/monitors/EjsSDJIBs61wKwycS6H4?&type=monitor&dataSourceId=`

With the alerting part the URL becomes this which links to the monitor details page
`http://localhost:5601/w/HMDqJO/app/alerting#/monitors/EjsSDJIBs61wKwycS6H4?&type=monitor&dataSourceId=`

This data source alerts card should only be present when workspaces is enabled which is only shown when the new home page settings is enabled?

Reply:
Changed the id to `MONITORS_NAV_ID` instead of `"alerting"`